crates/rpc/src/v1/impls/signer.rs

```diff
@@ -124,8 +124,8 @@ impl<D: Dispatcher + 'static> SignerClient<D> {
                             // Altering sender should always reset the nonce.
                             request.nonce = None;
                         }
-                        if let Some(gas_price) = modification.gas_price {
-                            request.gas_price = gas_price;
+                        if let Some(_gas_price) = modification.gas_price {
```

Comment:
```suggestion
                        if modification.gas_price.is_some() {
```

Sorry for the bikeshedding :)

Reply:
Nice one :)
Fixed.